docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -24,11 +24,26 @@ Servlet filter, otherwise it will run directly on top of Vert.x with no Servlet
 
 == Serving Static Resources
 
-To serve static resources you must place them in the `META-INF/resources` directory of your application. This location
+By default Quarkus will serve static files in the `META-INF/resources` directory of your application. This location
 was chosen as it is the standard location for resources in `jar` files as defined by the Servlet spec. Even though
 Quarkus can be used without Servlet following this convention allows existing code that places its resources in this
 location to function correctly.
 
+Additionally, Quarkus supports deployment of static files defined in a directory other than `META-INF/resources`.
+This is particularly useful when you want to leverage the power of a Quarkus backend alongside a frontend application (outside your Quarkus backend repository).
+In this workflow, you can configure your Quarkus backend using the configuration properties below to serve static files from the directory that contains your web application contents or the result of a frontend framework build process.
+
+[source,properties]
+----
+quarkus.static-resource.web.endpoint=/web <1>
```

Comment:
The main prefix is called `quarkus.http.root-path` so it's a bit unfortunate that we're not using the same term to designate both paths. I guess I would have used `prefix` instead of `endpoint` or `path`, myself.

Should we not use the quarkus.http.static` config prefix instead of `quarkus.static-resource`?

Reply:
Yes, I'll review the config knobs and update the change.